packages/eas-cli/src/commands/credentials.ts

```diff
@@ -9,6 +9,6 @@ export default class Credentials extends Command {
 
   async run() {
     const ctx = await createCredentialsContextAsync(process.cwd(), {});
-    await runStandaloneCredentialsManagerAsync(ctx, new SelectPlatform());
+    new CredentialsManager(ctx).runActionAsync(new SelectPlatform());
```

Comment:
```suggestion
    await (new SelectPlatform()).runAsync(ctx);
```
would this work as well? or were you planning to do this in a separate pr when you remove the credentials manager class?

Reply:
yes it would work, but mocking in tests would require a lot of changes. Currently, I can just mock one function and check what type was passed to runActionAsync, but with this approach I need to mock all Actions that potentially can be triggered from the action I'm currently testing